website/content/en/docs/faq.md

```diff
@@ -211,6 +211,8 @@ For information on upgrading Karpenter, see the [Upgrade Guide]({{< ref "./upgra
 
 ### How do I upgrade an EKS Cluster with Karpenter?
 
+Karpenter recommends that you always validate AMIs in your lower environments before using them in production environments. Read [Managing AMIs]({{<ref "./tasks/managing-amis" >}}) to understand best practices about upgrading your AMIs.
```

Comment:
I'm wondering if this FAQ entry should be rewritten entirely. It feels strange to point this out, and then continue to recommend an upgrade path that involves automatic upgrades. Should we rewrite this to detail an upgrade path which involves pinning to your current AMI, testing the new AMI in a lower environment, and bumping the pin?

Reply:
I view this more as a disclaimer, and then an explanation of how someone can do something, understanding the risks. I see it this way as I'm saying "to understand best practices", where the managing-ami doc talks about how to upgrade. Thoughts?

Reply:
That's fair, as a disclaimer does it make sense to put it in a note box? Something to separate it from the rest of the prose?